tests/sheet/sheetview_plugin.test.ts

```diff
@@ -1622,3 +1621,72 @@ describe("shift viewport up/down", () => {
     expect(model.getters.isVisibleInViewport({ sheetId, col, row })).toBeTruthy();
   });
 });
+
+describe.skip("smooth mode", () => {
```

Comment:
```suggestion
describe("smooth scrolling", () => {
```
or maybe it's not useful anymore since the other tests are all smooth now and needed to be adapted

Reply:
> maybe it's not useful anymore
correct 👍